### PR TITLE
Fix part of #1874: Fix span test for Home

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #1874 
Span test is Home moved to new `HomeSpanTest`

Screenshot 2020-11-23 at 11 35 11

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    }
+  }
+
+  @Config(qualifiers = "sw600dp-land")
+  @Test
+  fun testHomeSpanTest_checkSpanForItem0_landscape_tablet_hasCorrectSpanCount() {
+    launch<HomeActivity>(createHomeActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.home_recycler_view))
+        .check(
+          RecyclerViewMatcher.hasGridItemCount(
```
Comment (rt4914) [inline app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt]: Use direct import here and elsewhere. `hasGridItemCount(`

File: app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            spanCount = 3,
+            position = 0
+          )
+        )
+    }
+  }
+
+  @Config(qualifiers = "sw600dp-land")
```
Comment (rt4914) [inline app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt:122]: Add tests for `port` and `sw600dp-port` too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested changes. Thanks.

File: app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt]: done

File: app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt:122]: done

**Comment by rt4914** [comment]
I am getting this error of `No tests were` found. @anandwana001 
Screenshot 2020-11-23 at 12 01 03 PM

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**